packages/schematics/src/utils.ts

```diff
@@ -396,3 +418,15 @@ export function determineTargetProjectName(
   }
   return null;
 }
+
+/**
+ * Checking if the target project has e2e setup
+ * Method will check if angular project architect has e2e configuration to determine if e2e setup
+ */
+export function determineTargetProjectHasE2E(
+  angularJSON: any,
+  projectName: string,
+): boolean {
+  const { architect } = angularJSON.projects[projectName];
+  return Object.prototype.hasOwnProperty.call(architect, 'e2e');
```

Comment:
Nit: we're dealing with a very simple object here - with its origins in `.json` - and no prototypal chain to consider, so a simple `!!architect.e2e` or `Boolean(architect.e2e)` would have also been fine

Reply:
Got it, I simplified the determination logic. Thx